src/components/color-select/color-select.tsx

```diff
@@ -18,12 +19,15 @@ export const ColorSelect = ({ options, selectedName, onChange, hasError }: Color
         <div className={styles.root}>
             {options.map((o) => (
                 <button
+                    disabled={!o.inStock}
```

Comment:
it shouldn't be disabled.. user should still be able to select it, to check out images, price, etc.

Reply:
fixed